swiftgen-cli/images.swift

```diff
@@ -14,15 +14,13 @@ let imagesCommand = command(
   Option<String>("enumName", "Asset", flag: "e", description: "The name of the enum to generate"),
   Argument<Path>("DIR", description: "Directory to scan for .imageset files.", validator: dirExists)
 ) { output, templateName, templatePath, enumName, path in
-  let parser = AssetsCatalogParser()
-
-  switch path.extension {
-  case "xcassets"?:
-    parser.parseCatalog(at: path)
-  default:
+  guard let `extension` = path.extension, `extension` == "xcassets" else {
```

Comment:
Haha I was more thinking about `guard case "xcasstes" = path.extension` but that works too ^^